apidoc/Titanium/UI/TabGroup.yml

```diff
@@ -199,6 +199,10 @@ methods:
         type: disableTabOptions
     platforms: [android]
     since: 3.6.0
+    deprecated:
+        since: "12.7.0"
+        removed: "13.0.0"
```

Comment:
We usually only add the removed tag once the release that removed it is actually out, so I'd suggest to remove it and set a reminder to when master will become 13.0.0 (we can plan it as the next release after the 12.7.0 release). Then, you can do a PR with the removal itself and the docs for it.